apps/dev/components/header.js

```diff
@@ -108,6 +108,11 @@ export default function Header() {
               <a>Middleware protected</a>
             </Link>
           </li>
+          <li className={styles.navItem}>
```

Comment:
Is this just for testing? If so, I don't think we need it here, as we can just append the param to the URL `/middleware-protected` manually. Please remove 🙏

Reply:
Sure.